fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -2834,13 +2844,15 @@ private void createOlapTable(Database db, CreateTableStmt stmt) throws UserExcep
                                 TimeUtils.getCurrentFormatTime());
             }
         } catch (DdlException e) {
+            LOG.warn("create table failed {} - {}", tabletIdSet, e.getMessage());
             for (Long tabletId : tabletIdSet) {
                 Env.getCurrentInvertedIndex().deleteTablet(tabletId);
             }
             // only remove from memory, because we have not persist it
             if (Env.getCurrentColocateIndex().isColocateTable(tableId)) {
                 Env.getCurrentColocateIndex().removeTable(tableId);
             }
+            dropTable(db, tableId, true, 0L);
```

Comment:
if table has not created,  dropTable will throws a exception.  here shoud ignore the dropTable exception， otherwise it will throw the dropTable exception.

Reply:
fixed